scripts/setup_osx.sh

```diff
@@ -1,38 +1,265 @@
+BASH_PROF="${HOME}/.bash_profile"
+
+# Install dependencies
+echo "Installing garage dependencies"
+echo "You will probably be asked for your sudo password"
+
+# Homebrew is required first to install the other dependencies
+hash brew 2>/dev/null || {
+  # Install the Xcode Command Line Tools
+  xcode-select --install
+  # Install Homebrew
+  /usr/bin/ruby -e "$(curl -fsSL \
+    https://raw.githubusercontent.com/Homebrew/install/master/install)"
+}
+
+# For installing garage: bzip2, git, glfw, unzip, wget
+# For building glfw: cmake
+# Required for OpenAI gym: cmake boost boost-python ffmpeg sdl2 swig wget
+# Required for OpenAI baselines: cmake openmpi
+brew update
+brew install \
+  bzip2 \
+  git \
+  glfw \
+  unzip \
+  wget \
+  cmake \
+  boost \
+  boost-python \
+  ffmpeg \
+  sdl2 \
+  swig \
+  openmpi
+
+# Leave a note in ~/.bashrc for the added environment variables
+if [[ "${_arg_set_envvar}" = on ]]; then
+  echo -e "\n# Added by the garage installer" >> "${BASH_PROF}"
+fi
+
+# Set up MuJoCo
+if [[ ! -d "${HOME}"/.mujoco/mjpro150 ]]; then
+  mkdir "${HOME}"/.mujoco
+  MUJOCO_ZIP="$(mktemp -d)/mujoco.zip"
+  wget https://www.roboti.us/download/mjpro150_osx.zip -O "${MUJOCO_ZIP}"
+  unzip -u "${MUJOCO_ZIP}" -d "${HOME}"/.mujoco
+else
+  print_warning "MuJoCo is already installed"
+fi
+# Configure MuJoCo as a shared library
+export LD_LIBRARY_PATH="$LD_LIBRARY_PATH:${HOME}/.mujoco/mjpro150/bin"
+LD_LIB_ENV_VAR="LD_LIBRARY_PATH=\"\$LD_LIBRARY_PATH:${HOME}/.mujoco/mjpro150"
+LD_LIB_ENV_VAR="${LD_LIB_ENV_VAR}/bin\""
+if [[ "${_arg_set_envvar}" = on ]]; then
+  echo "export ${LD_LIB_ENV_VAR}" >> "${BASH_PROF}"
+fi
+
+# Set up conda
+hash conda 2>/dev/null || {
+  CONDA_INSTALLER="$(mktemp -d)/miniconda.sh"
+  sudo chmod u+x "${CONDA_INSTALLER}"
+  wget https://repo.continuum.io/miniconda/Miniconda2-latest-MacOSX-x86_64.sh \
+    -O "${CONDA_INSTALLER}"
+  sudo bash "${CONDA_INSTALLER}" -b
+  # Add conda to executable programs
+  CONDA_PATH="${HOME}/miniconda2/bin/"
+  export PATH="$PATH:${CONDA_PATH}"
+  PATH_ENV_VAR="PATH=\"\$PATH:${CONDA_PATH}\""
+  if [[ "${_arg_set_envvar}" = on ]]; then
+    echo "export ${PATH_ENV_VAR}" >> "${BASH_PROF}"
+  fi
+}
+conda update -q -y conda
+
+# Create conda environment
 conda env create -f environment.yml
-conda env update
+if [[ "${?}" -ne 0 ]]; then
+  print_error "Error: conda environment could not be created"
+fi
+
+# Extras
+source activate garage
+# Prevent pip from complaining about available upgrades
+pip install --upgrade pip
+# Fix Box2D install
+# See https://github.com/openai/gym/issues/100
+# See https://github.com/pybox2d/pybox2d/issues/82
+pip uninstall -y Box2D Box2D-kengz box2d-py
+pip install Box2D
+# We need a MuJoCo key to import mujoco_py
+cp ${_arg_mjkey_path} "${HOME}"/.mujoco/mjkey.txt
+python -c 'import mujoco_py'
+# Set up pre-commit in local repo
+pre-commit install -t pre-commit
+pre-commit install -t pre-push
+pre-commit install -t commit-msg
+source deactivate
```

Comment:
Copy the `config_personal_template.py` to `config_personal.py` so that users don't see the nag message on their first attempt to run something.